.gitignore

```diff
@@ -19,3 +19,4 @@ vite.config.js.timestamp-*
 .test-tmp
 symlink-from
 .idea/
+playgrounds
```

Comment:
This doesn't work once the files are already in the index.

Reply:
Damnit you're right! I'll take another look at the previous way